next.config.js

```diff
@@ -248,25 +248,11 @@ async function redirects() {
       destination: '/references/faqs',
       permanent: true,
     },
-    { source: '/org/token.html', destination: '/ecosystem/stacks-token', permanent: true },
     {
       source: '/org/whitepaper-blockchain.html',
       destination: 'https://blockstack.org/whitepaper.pdf',
       permanent: true,
     },
-    { source: '/org/wallet-intro.html', destination: '/stacks-wallet/overview', permanent: true },
-    { source: '/org/wallet-install.html', destination: '/stacks-wallet/install', permanent: true },
-    { source: '/org/wallet-use.html', destination: '/stacks-wallet/usage', permanent: true },
-    {
-      source: '/org/wallet-troubleshoot.html',
-      destination: '/stacks-wallet/troubleshooting',
-      permanent: true,
-    },
-    {
-      source: '/org/tokenholders.html',
-      destination: '/ecosystem/stacks-token-holders',
-      permanent: true,
-    },
```

Comment:
@agraebe we can't just remove redirects. We should always keep them, and redirect them to somewhere new. Ideally we'd avoid a 404 page. Can you please update this and have it redirect either to the homepage, or a page dedicated to the wallet?

Reply:
great catch! updating that now. thanks for reviewing the PRs @aulneau!